docs/client-sdk/javascript/tutorials/use-content-types.mdx

```diff
@@ -61,58 +61,148 @@ To do this, pass the `CompositeCodec` content type as an option to the `Client.c
+## Display messages with a specific content type
 
-2. Send a message using the specified custom content type and fallback plain text.
+The `contentTypeId` will be available to help distinguish the content type the message contains. 
 
-    ```typescript
-    // Assuming NumberCodec can be used to encode numbers and is
-    // identified with ContentTypeNumber, you can use it as follows.
-    conversation.send(3.14, {
-      contentType: ContentTypeNumber,
-      contentFallback: '3.14'
-    })
-    ```
+For example, if the content is an audio type, then: 
 
-    This example sends a message using the fictional "number" custom content type, including fallback plain text.
+```javascript
+const contentTypeId = message.contentType.typeId
+// contentTypeId is `audio-key`
+```
 
-    The receiving client uses the `contentType` value of `ContentTypeNumber` to identify the content type of the `3.14` message sent through the `send` API. If the receiving client supports the content type, it displays the message. If it doesn't support the content type, it displays the fallback plain text.
+Let's say your app receives a message tagged as `audio-key` in the `contentTypeId` and this content type uses the `AudioCodec` for encoding and decoding content. To display the audio message in your app, write a conditional in the view layer. 
+
+For example, in a `messageList.tsx` file that displays messages, you can do the following:
+
+```jsx
+const TypeOfMessage = ({ message }: MessageTileProps): JSX.Element => {
+  const contentTypeId = message.contentType.typeId;
+  const isAudioMessage = contentTypeId === 'audio-key';
+
+  if (isAudioMessage) {
+    return (
+      <audio controls>
+        <source src={message.content} type="audio/mpeg" />
+      </audio>
+    );
+  } else if (message.error) {
+    return <div>{message.error.message}</div>;
+  } else {
+    return <Emoji text={message.content || ''} />;
+  }
+};
+```
 
-    To learn more about the `send` API, see [Sending messages](quickstart#sending-messages).
-    <!--reference docs are now missing the pre-existing sendMessage topic: https://xmtp.org/docs/client-sdk/javascript/reference/classes/Client#sendmessage. We have an open eng issue to fix this. In the meantime,linking here.-->
+If your app is unable to handle a custom content type, then the  `contentFallback` option will automatically display. 
 
-To learn more about sending custom content types, see [Different content types](https://github.com/xmtp/xmtp-js/blob/4157fadd80bce80c8094135f3e47d3856515468f/README.md#different-types-of-content).
+To learn more from an example implementation, see this [voice-note branch](https://github.com/xmtp-labs/xmtp-inbox-web/compare/dev...demo/content-type/voice-note) in the XMTP Inbox repo.
```

Comment:
Just curious if you want to put this in a PR in xmtp-inbox-web so that people can try using audio content type in the app preview? You worked so hard on it people should be able to try it! And it would be a great accompaniment to this tutorial you wrote! You can open the PR and then close it to ensure that it doesn't get merged.

Reply:
I actually had a small convo with Daria about opening a PR. I dont think it is currently ready. This branch is basically a rough implementation of adding a custom content type. I think this branch will be obsolete when Pat's remote attachment is added to xmtp-inbox-web.